README.md

```diff
@@ -202,16 +202,10 @@ nvm install
 nvm use
 ```
 
-Install yarn with:
-
-``` shell
-npm i -g yarn
-```
```

Comment:
This will need to be moved to line 499 because we now cannot assume yarn is installed for that part